examples/chef/create_docker.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e # Exit on any script error.
+
+IMAGE_NAME='us.gcr.io/nest-eng/chef/dimmable-light'
```

Comment:
This seems a bit hardcoded:

- why is the image nest-eng? Can we add it as an argument (so at least cloudbuild sets it but we make the script generic enough to be an OSS component)?
- file requires a license blurb (see other shell scripts)
- same comment regarding hardcoding for /workspace and the name of linux_..... .

Reply:
Removed the hardcoding from this script and made use of Substitution Variables from the Google Cloud build to make this more generic.